app/views/finders/_select_facet.html.erb

```diff
@@ -6,7 +6,8 @@
     :aria_controls_id => "js-search-results-info",
     :options_container_id => select_facet.key,
     :options => select_facet.options("js-search-results-info", select_facet.key),
-    :closed_on_load => close_facet
+    :closed_on_load => close_facet,
+    :show_filter => true
```

Comment:
I know why you added this but we should remove it.

Reply:
Thanks, good spot. I added it so I could test all of this in the browser in a finder, not just in the component guide.